algorithmic_efficiency/workloads/librispeech_deepspeech/librispeech_jax/models.py

```diff
@@ -554,7 +554,7 @@ def __call__(
                 deterministic=deterministic)
         outputs = jnp.concatenate([outputs, backward_outputs], axis=-1)
         cell_idx += 1
-
+      outputs = outputs * (1 - input_paddings[:, :, None])
```

Comment:
why this change in jax deepspeech model ?

Reply:
Mainly because they help in easily matching pytorch and jax outputs. They correspond to the padded out portions of the outputs and aren't used by any later layers.

Reply:
can you modify pytorch code instead to remove this line ?

Reply:
I have removed this line! I will later modify the test script to only consider non-padded outputs.